barretenberg/cpp/src/barretenberg/relations/logderiv_lookup_relation.hpp

```diff
@@ -32,11 +32,11 @@ template <typename FF_> class LogDerivLookupRelationImpl {
         3, // log derivative lookup argument sub-relation
     };
 
-    // TODO(https://github.com/AztecProtocol/barretenberg/issues/1036): Scrutinize these adjustment factors. Counting
-    // degrees suggests the first subrelation should require an adjustment of 2.
+    // Note: the required correction for the second sub-relation is technically +1 but the two corrections must agree
+    // due to the way the relation algebra is written so both are set to +2.
     static constexpr std::array<size_t, 2> TOTAL_LENGTH_ADJUSTMENTS{
```

Comment:
didn't know we changed to this length adjustments style of specifying relation degree

Reply:
This is how its always been as far I knew. I'm not sure why they opted to specify this as an 'adjustment' vs just the PG relation degree